### PR TITLE
Removed notices that revealed themselfes over night ...

### PR DESCRIPTION
**Pull request by annando**

### CODE REVIEW

File: src/Core/NotificationsManager.php [hunk 1]
```diff
@@ -344,6 +344,10 @@ private function formatNotifs(array $notifs, $ident = "")
 						break;
 
 					case ACTIVITY_FRIEND:
+						if (!isset($it['object'])) {
```
Comment (MrPetovan) [inline src/Core/NotificationsManager.php]: Could it be related to the fact that the base item table doesn't use `contact` anymore?

File: src/Network/Probe.php [hunk 2]
```diff
@@ -933,7 +933,12 @@ public static function profile($profile_link)
 		}
 
 		$prof_data = [];
-		$prof_data["addr"]         = $data["addr"];
+
+		// The "addr" is not always part of the fetched data
+		if (!empty($data["addr"])) {
+			$prof_data["addr"] = $data["addr"];
```
Comment (MrPetovan) [inline src/Network/Probe.php]: This means the `addr` key of the `$prof_data` array may or may not be set from now on. This is inconsistent with the previous behavior which would have left `$prof_data["addr"]` set to `null` by default if `$data["addr"]` wasn't set. Please use `defaults()` instead.

**Comment by MrPetovan** [comment]
Part of #5470

File: src/Core/NotificationsManager.php (hunk 1 above)
Comment (annando) [reply src/Core/NotificationsManager.php]: We will see what's the root of the problem is.

File: src/Network/Probe.php (hunk 2 above)
Comment (annando) [reply src/Network/Probe.php]: I'm unsure if we possibly would overwrite an existing "addr" with an empty value by using "defaults".

Comment (MrPetovan) [reply src/Network/Probe.php]: No, the `$prof_data` array is initialized on line 935, there's no prior value.

**Pull request merged**